R/ospsuite-env.R

```diff
@@ -27,7 +27,7 @@ ospsuiteEnv$numberOfCores <- function() {
 ospsuiteEnv$showProgress <- FALSE
 
 # Specifies the symbol used for µ. This will be set by the .NET layer
```

Comment:
_µ_ in the comment line above is OK for R CMD check?

Reply:
Apparently... yes: https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-r-r99ji/builds/48493684#L1205

Reply:
hmm, ok